ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/DefaultTransform.kt

```diff
@@ -111,6 +112,28 @@ public fun HttpClient.defaultTransformers() {
                 proceedWith(HttpResponseContainer(info, response.status))
             }
 
+            MultiPartData::class -> {
+                val contentType =
+                    context.response.contentType() ?: throw IllegalStateException("No content type in response")
+                if (!contentType.match(ContentType.MultiPart.FormData)) {
+                    throw IllegalStateException(
+                        "Expected multipart/form-data, got $contentType"
+                    )
+                }
+
+                val rawContentType = context.response.headers[HttpHeaders.ContentType]
+                    ?: error("No content type provided for multipart")
```

Comment:
Feels redundant as we already have `contentType`

Reply:
but it's string, I want to avoid serialization